spark/src/test/scala/org/apache/spark/sql/delta/DeltaInsertIntoColumnOrderSuite.scala

```diff
@@ -145,7 +149,9 @@ class DeltaInsertIntoColumnOrderSuite extends DeltaInsertIntoTest {
       insertData = TestData("a long, s struct <y int, x: int>",
         Seq("""{ "a": 1, "s": { "y": 5, "x": 4 } }""")),
       expectedResult = ExpectedResult.Success(expectedAnswer),
-      includeInserts = inserts -- insertsDataframe.intersect(insertsByName)
+      // Exclude dataframe inserts by name (except streaming) which don't support implicit cast.
+      // See negative test below.
+      includeInserts = inserts -- (insertsByName.intersect(insertsDataframe) - StreamingInsert)
```

Comment:
`inserts` is one of:
-  `insertsAppend - StreamingInsert`
- `insertsOverwrite - SQLInsertOverwritePartitionByPosition`
- `Set(StreamingInsert)`
- `Set(SQLInsertOverwritePartitionByPosition)`

It doesn't always contain `StreamingInsert`, we don't want to add it then otherwise we'll get duplicate tests